Changelog.txt

```diff
@@ -3705,4 +3705,8 @@ Additionally, the problem of zig-zag issue following in the South direction has
 	- Fishing skill in default script pack should updated.
 	- Delay should set to 2.5-3.0
 	- All softcoded system should removed.
-	- If anyone want to code their own fishing system, they can use SKF_SCRIPTED.
\ No newline at end of file
+	- If anyone want to code their own fishing system, they can use SKF_SCRIPTED.
+- Added: EF_DAMAGETOOLS flag implemented to the sphere.
+	- Added LOCAL.ITEMDAMAGECHANCE to the @Success trigger for skills to decide damage chance for tools. (Default: 25)
+	- Added LOCAL.ITEMDAMAGEAMOUNT to the @Success trigger for skills to decide damage amount. (Default: 1)
+	- EF_DAMAGETOOLS only affects the skills with SKF_GATHER flag.
```

Comment:
Are we sure to only have that with gather skills? I can't imagine different scenarios right now but i'm just asking for ideas

Reply:
Honestly, I thought a bit about it but it seems like only craft and gather skills use tools, only Magery uses spellbook but it's not a tool. Some skills like Taming uses custom Taming Staffs etc but they are not default, so they can add damage if they want in their scripts.